src/manipulation.js

```diff
@@ -0,0 +1,7 @@
+var rxhtmlTag = /<(?!area|br|col|embed|hr|img|input|link|meta|param)(([a-z][^\/\0>\x20\t\r\n\f]*)[^>]*)\/>/gi;
+
+jQuery.restoreLegacyInsecureHtmlPrefilter = function() {
```

Comment:
Optionally, to make people really think before invoking that, we can do:
```suggestion
jQuery.UNSAFE_restoreLegacyInsecureHtmlPrefilter = function() {
```

Reply:
Maybe `UNSAFE_restoreHtmlPrefilter`?

Reply:
@dmethvin That seems to suggest to me that we don't have `htmlPrefilter` now but we do, just with a different value.

Reply:
How about just `jQuery.UNSAFE_restoreLegacyHtmlPrefilter` since we have UNSAFE in the name now.

Reply:
PR updated.